geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1513,21 +1513,24 @@ public Object selectValue(String queryPredicate) throws FunctionDomainException,
    * cleared from the client and current value for this key is inserted into the local cache before
    * this call returns.
    *
-   * @param key The key on which to register interest. If the key is a <code>List</code>, then all
-   *        the keys in the <code>List</code> will be registered. The key can also be the special
-   *        token 'ALL_KEYS', which will register interest in all keys in the region. In effect,
-   *        this will cause an update to any key in this region in the CacheServer to be pushed to
-   *        the client.
+   * @param key The key on which to register interest.
+   *
+   *        The following <code>List</code> and 'ALL_KEYS' behavior is now deprecated: If the key is
+   *        a <code>List</code>, then all the keys in the <code>List</code> will be registered. The
+   *        key can also be the special token 'ALL_KEYS', which will register interest in all keys
+   *        in the region. In effect, this will cause an update to any key in this region in the
+   *        CacheServer to be pushed to the client.
    *
    *        <p>
    *        This method uses the default <code>InterestResultPolicy</code>.
    *        </p>
    *
    *        <p>
-   *        <i>Using 'ALL_KEYS' is the same as calling {@link #registerInterestRegex(String)} with
+   *        <i> Using 'ALL_KEYS' is the same as calling {@link #registerInterestRegex(String)} with
    *        ".*" as the argument. This means that all keys any type are pushed to the client and
    *        inserted into the local cache.</i>
    *        </p>
+   *        End of deprecation
```

Comment:
Do you want to say what API to use instead of this one for the deprecated cases?